sample/test-proguard-rules.pro

```diff
@@ -1,3 +1,2 @@
 -dontwarn org.junit.**
--dontwarn android.test.**
--dontwarn android.support.test.**
+-dontwarn android.**
```

Comment:
There were some other proguard warnings and just decided to ignore entire android namespace. Tests are passing.